app/src/main/java/mozilla/lockbox/presenter/EditItemPresenter.kt

```diff
@@ -127,8 +129,18 @@ class EditItemPresenter(
             .addTo(compositeDisposable)
 
         view.togglePasswordClicks
-            .map { ItemDetailAction.SetPasswordVisibility(view.isPasswordVisible.not()) }
-            .subscribe(dispatcher::dispatch)
+            .subscribe {
+                dispatcher.dispatch(
+                    ItemDetailAction.SetPasswordVisibility(view.isPasswordVisible.not())
+                )
+                view.setTextSelectionOnPasswordToggle()
+            }
+            .addTo(compositeDisposable)
+
+        view.usernameFieldClicks
+            .subscribe {
+                view.setTextSelectionOnPasswordToggle()
```

Comment:
What is this doing? This looks weird. Could you comment it?

Not sure it's quite as useful as we might think.